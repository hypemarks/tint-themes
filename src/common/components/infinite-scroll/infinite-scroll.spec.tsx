import { render } from '@testing-library/react';
import React from 'react';
import InfiniteScroll from './infinite-scroll';

describe('Test <InfiniteScroll /> Component', () => {
  it('should render snapshot', () => {
    const wrapper = render(<InfiniteScroll fetchNextPage={() => new Promise(jest.fn)} />);
    expect(wrapper.asFragment()).toMatchSnapshot();
  });
});
