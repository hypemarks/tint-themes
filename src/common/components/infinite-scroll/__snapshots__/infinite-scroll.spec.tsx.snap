// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <InfiniteScroll /> Component should render snapshot 1`] = `
<DocumentFragment>
  <div
    class="infinite-scroll false"
    data-testid="infinite-scroll"
  />
</DocumentFragment>
`;
